### PR TITLE
Fix for #41

### PR DESCRIPTION
**Pull request by lowerends**
The Blade extension `is` now also accepts an array of roles and returns true if the user has any of the given roles.

### CODE REVIEW

**Comment by lowerends** [comment]
Can you give an update on this, please?

File: src/Defender/Traits/HasUserRolesTrait.php [hunk 1]
```diff
@@ -39,6 +39,28 @@ public function hasRole($role)
...
     /**
+     * Returns true if the given user has any of the given roles.
+     *
+     * @param $roles
+     *
+     * @return bool
+     */
+    public function hasRoles($roles)
+    {
+        if (is_array($roles)) {
```
Comment (vluzrmos) [inline src/Defender/Traits/HasUserRolesTrait.php]:
```php
$roles = is_array($roles) ? $roles : fung_get_args();
```
will be better in cases the user do:
@is('admin', 'client', 'guest')

File: src/Defender/Traits/HasRolePermissionsTrait.php [hunk 2]
```diff
@@ -19,11 +19,11 @@
...
     {
-        $permissionModel     = config('defender.permission_model');
+        $permissionModel = config('defender.permission_model');
         $permissionRoleTable = config('defender.permission_role_table');
-        $roleKey             = config('defender.role_key');
-        $permissionKey       = config('defender.permission_key');
-        
+        $roleKey = config('defender.role_key');
+        $permissionKey = config('defender.permission_key');
+
```
Comment (vinicius73) [inline src/Defender/Traits/HasRolePermissionsTrait.php]: fix identation
these lines were not modified.

File: src/Defender/Traits/HasUserRolesTrait.php (hunk 1 above)
Comment (vinicius73) [inline src/Defender/Traits/HasUserRolesTrait.php]: It would be much better

**Comment by vinicius73** [comment]
The proposal could now be accepted, but would be very nice if you could make these modifications.

**Comment by vluzrmos** [comment]
And... thanks for that PR.

File: src/Defender/Traits/HasUserRolesTrait.php (hunk 1 above)
Comment (lowerends) [inline src/Defender/Traits/HasUserRolesTrait.php]: Agree, will fix it.

File: src/Defender/Traits/HasRolePermissionsTrait.php (hunk 2 above)
Comment (lowerends) [inline src/Defender/Traits/HasRolePermissionsTrait.php]: Indeed, but then StyleCI is giving an error on those lines, that's why I fixed them.

**Comment by lowerends** [comment]
@vluzrmos @vinicius73 Thanks for the feedback, and you're welcome :)

File: src/Defender/Traits/HasRolePermissionsTrait.php (hunk 2 above)
Comment (vinicius73) [inline src/Defender/Traits/HasRolePermissionsTrait.php]: I understand , a real shame.
We que configure it better.

**Comment by vinicius73** [comment]
Thanks for the collaboration

**Pull request merged**